utils.py

```diff
@@ -31,12 +29,19 @@ def take_screenshot():
 
 
 def prepare_image(img):
+    import wx
+    wx.App()
```

Comment:
Nice idea! that actually sounds like the right abstraction to me and it solves the screen grab coordinates issue for the record.py app as well :tada: